packages/server/src/server.ts

```diff
@@ -47,14 +49,18 @@ function verifyToken(request: Request, response: Response, next: NextFunction) {
 
 app.use(verifyToken)
 
+const dbConnection = DbConnection.getInstance()
+dbConnection.postgresConnection = postgresConnection
+dbConnection.dbConnection = databaseConnection
+dbConnection.pgp = pgPromise()
 // Must be first to initialize
 initializeCommons(pgp)
-registerDBAdmin(app, dbConnection, postgresConnection, pgp)
-registerBulkApi(app, dbConnection, pgp)
-registerInspection(app, dbConnection, pgp)
-registerAdditionalApi(app, dbConnection, pgp)
-registerLanguagesApi(app, dbConnection, pgp)
-registerHistoryApi(app, dbConnection, pgp)
+registerDBAdmin(app, DbConnection.getInstance(), postgresConnection, pgp)
```

Comment:
Should this and the other all get `dbConnection`?

Reply:
You are quite right, I forgot to use dbConnection in some API's. Meaning that the one not using dbConnection won't work correctly for multiple repositories!
Will make changes to fix this.